F1WM/Repositories/CalendarRepository.cs

```diff
@@ -21,7 +21,7 @@ public async Task<Calendar> GetCalendar(int year)
 			var dbRace = await context.Races
 				.Include(r => r.FastestLap).ThenInclude(f => f.Entry).ThenInclude(e => e.Car)
 				.Include(r => r.FastestLap).ThenInclude(f => f.Entry).ThenInclude(e => e.Driver).ThenInclude(d => d.Nationality)
-				.Where(r => r.FastestLap.Frlpos == "1")
+				.Where(r => r.FastestLap.Frlpos == null || r.FastestLap.Frlpos == "1")
```

Comment:
This is not related to the bug and doesn't actually resolve the problem with calendar. The problem here lies actually in an incorrect mapping - Race has many FastestLap entities connected to it, so FastestLap property should have its type changed to IEnumerable<FastestLap> and its name changed accordingly.